payload/wrapper.go

```diff
@@ -25,3 +94,24 @@ func Base64EncodeForPHPEval(cmd string) string {
 
 	return fmt.Sprintf(`<?php eval(base64_decode("%s")); ?>`, cmd64)
 }
+
+// Creates a valid PHP filter string from an input. Normally these are PHP payloads, but there are
+// exceptions.
+func PHPFilter(chain string) string {
```

Comment:
I think I'd just include the ambionics write (or any type of information on when you might use this). This was admittedly new to me so a citation would have been useful.

Reply:
Added a comment to the oldest information about the technique I could find (2018), renamed it to `PHPIconvFilter` as we may want to add more (like the ambionics one), and made added RFC 3501 and URL safe base64 variants in the maps in case it is ever needed.